### PR TITLE
New code coverage filters

### PR DESCRIPTION
**Pull request by webreidi**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@Mikejo5000 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/test/using-code-coverage-to-determine-how-much-code-is-being-tested.md [hunk 1]
```diff
@@ -73,6 +73,20 @@ You can also have the results displayed for lines by choosing **Add/Remove Colum
 > [!TIP]
 > A line of code can contain more than one code block. If this is the case, and the test run exercises all the code blocks in the line, it is counted as one line. If some but not all code blocks in the line are exercised, it is counted as a partial line.
 
+## Filtering code coverage results
```
Comment (Mikejo5000) [inline docs/test/using-code-coverage-to-determine-how-much-code-is-being-tested.md:76]:
```suggestion
## Filter code coverage results
```

File: docs/test/using-code-coverage-to-determine-how-much-code-is-being-tested.md [hunk 2]
```diff
@@ -73,6 +73,20 @@ You can also have the results displayed for lines by choosing **Add/Remove Colum
...
+
+The **Code Coverage Results** window usually shows the result for the entire solution. The results can be filtered to show the results for only the files that have been updated in the current branch. 
+
+- **To view the changeset Report**, select the Configure Code Coverage Views icon in the Code Coverage Results Window. Then select "Changeset Report" from the Report Content drop-down. Update the Active Repository and Base Branch to compare against for it to provide an the comparison report.
+
+From the search box in the **Code Coverage Results** window, there are several ways the reprt can be filtered.
```
Comment (Mikejo5000) [inline docs/test/using-code-coverage-to-determine-how-much-code-is-being-tested.md:82]:
```suggestion
From the search box in the **Code Coverage Results** window, there are several ways the report can be filtered.
```

**Review by Mikejo5000 — DISMISSED** [review]
Thanks! I left a few minor style guide edits.

**Comment by webreidi** [comment]
@Mikejo5000 Thank you. Changes made and committed.

File: docs/test/using-code-coverage-to-determine-how-much-code-is-being-tested.md [hunk 3]
```diff
@@ -73,6 +73,20 @@ You can also have the results displayed for lines by choosing **Add/Remove Colum
 > [!TIP]
 > A line of code can contain more than one code block. If this is the case, and the test run exercises all the code blocks in the line, it is counted as one line. If some but not all code blocks in the line are exercised, it is counted as a partial line.
 
+## Filter code coverage results
+
+The **Code Coverage Results** window usually shows the result for the entire solution. The results can be filtered to show the results for only the files that have been updated in the current branch. 
+
+- **To view the changeset Report**, select the Configure Code Coverage Views icon in the Code Coverage Results Window. Then select "Changeset Report" from the Report Content drop-down. Update the Active Repository and Base Branch to compare against for it to provide an the comparison report.
```
Comment (Jak-MS) [inline docs/test/using-code-coverage-to-determine-how-much-code-is-being-tested.md:80]:
```suggestion
- **To view the changeset Report**, select the **Configure Code Coverage Views** icon in the **Code Coverage Results** window. Then select **Changeset Report** from the **Report Content** drop-down. Update the Active Repository and Base Branch to compare against for it to provide the comparison report.
```

**Review by Jak-MS — APPROVED** [review]

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**